core/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ option(BUILD_CORE_SAMPLES "Build cubos core samples" OFF)
 option(BUILD_CORE_TESTS "Build cubos core tests?" OFF)
 option(BUILD_CORE_SHARED "Build cubos core as shared library?" ON)
 
+option(ENABLE_PROFILING "Should profiling macros be compiled?" ON)
+set(CUBOS_METRIC_MAX_ENTRIES "1024" CACHE STRING "Maximum number of entries each metric can store")
```

Comment:
why was 1024 chosen as the max number of entries? Is this arbitrary?

Reply:
Just a dummy value. It can be changed easily through the method `setMaxEntries`